src/lib/tzkt/api.ts

```diff
@@ -0,0 +1,218 @@
+import axios, { AxiosError } from 'axios';
+
+import { TempleChainId } from 'lib/temple/types';
+
+import {
+  TzktOperation,
+  TzktOperationType,
+  TzktQuoteCurrency,
+  TzktAccountTokenBalance,
+  allInt32ParameterKeys,
+  TzktGetRewardsParams,
+  TzktGetRewardsResponse,
+  TzktRelatedContract
+} from './types';
+
+//// CHAIN ID
+
+const TZKT_API_BASE_URLS = {
+  [TempleChainId.Mainnet]: 'https://api.tzkt.io/v1',
+  [TempleChainId.Jakartanet]: 'https://api.jakartanet.tzkt.io/v1',
+  [TempleChainId.Ghostnet]: 'https://api.ghostnet.tzkt.io/v1',
+  [TempleChainId.Dcp]: 'https://explorer.tlnt.net:8001/v1',
+  [TempleChainId.DcpTest]: 'https://explorer.tlnt.net:8009/v1'
+};
+
+export const TZKT_API_BASE_URLS_MAP = new Map(Object.entries(TZKT_API_BASE_URLS) as [TempleChainId, string][]);
```

Comment:
what is the point of the Map here?   mb lets use hardcoded TZKT_API_BASE_URLS Record? (just add types for it)